python/seldon_core/microservice.py

```diff
@@ -322,7 +328,7 @@ def rest_prediction_server():
                 )
                 StandaloneApplication(app, user_object, options=options).run()
 
-            logger.info("REST gunicorn microservice running on port %i", port)
+            logger.info("REST microservice running on port %i", port)
```

Comment:
I believe this logger was correct. In case of many workers we do run with the gunicorn.

Reply:
Fixed.